contrib/pyln-testing/pyproject.toml

```diff
@@ -17,7 +17,7 @@ ephemeral-port-reserve = "^1.1.4"
 psycopg2-binary = "^2.9.3"
 python-bitcoinlib = "^0.11.0"
 jsonschema = "^4.4.0"
-pyln-client = "^0.11"
+pyln-client = ">=0.12.1"
```

Comment:
This is the minimal version required so we have `LightningRpc.get_json_id`. Bound expressed as lower bound, so we don't have to bump this every single time.